### DISCUSSION TITLE
Other Frequencies in Listen All Mode

### DISCUSSION THREAD

**Discussion by tfc87** [allangood/rtlamr2mqtt#229] [Q&A]
First of all, thank you for the great work here...I've been able to pull my water meter information in pretty easily!  I'm still hoping to get my power meter (FlexNet 530x) and gas meter (iTron B172) working but starting to wonder if they're reporting at a different frequency.  When I run the container in Listen All Meters mode, it appears it defaults to 912380000.

Does anyone know how to run the Listen All mode at other frequencies to see if I can track down the additional meters?  Thank you in advance for your help!

**Comment by jbarneson** [comment]
i'm in the same boat -- everything up and running easy (thanks!) -- but no data from my flexnet gas and water meters.. i am seeing other meters (maybe electric) with ID's that don't match up with my meters..  I know my water meters are broadcasting on 415mhz

I thought maybe the -f custom_param would get me there, but i'm not finding any devices.. ([ref docs](https://inst.eecs.berkeley.edu/~ee123/sp14/rtl_sdr_install.html))

`rtltcp: "-s 2048000 -f 45100000"
rtlamr: "-unique=true"`

**Comment by jbarneson** [comment]
i did find this in issues this morning: https://github.com/allangood/rtlamr2mqtt/issues/153

and updated my config to: 

`rtltcp: "-s 2048000"
rtlamr: "-unique=true -centerfreq=451000000"
`

**Reply by tfc87** [reply]
It's been a few months since I've had a chance to circle back on this and try to figure it out.  Thank you for your responses!  Do you mind elaborating on where you made these config changes?  I was trying to run it under the Listen All Meters mode at different frequencies to see if I can sniff out my other meters.  I have a docker compose for rtllisten.
